torch/backends/mkl/__init__.py

```diff
@@ -45,12 +45,12 @@ def __init__(self, enable):
     def __enter__(self):
         if self.enable == VERBOSE_OFF:
             return
-        st = torch._C._verbose.mkl_set_verbose(self.enable)
+        st = torch._C._verbose.mkl_set_verbose(self.enable)  # type: ignore[attr-defined]
```

Comment:
```
  Error (MYPY) [attr-defined]
    Module has no attribute "_verbose"

        52  |        return self
        53  |
        54  |    def __exit__(self, exc_type, exc_val, exc_tb):
    >>> 55  |        torch._C._verbose.mkl_set_verbose(VERBOSE_OFF)
        56  |        return False
```

I think we could (a) ignore this lint error or (b) tell mypy to pick-up the [_verbose.pyi](https://github.com/pytorch/pytorch/blob/main/torch/_C/_verbose.pyi) stub file.

Reply:
We can do (b) via `import torch._C._verbose as _verbose`

Reply:
Yeah we should do b)